react/src/components/Utils/Utils.js

```diff
@@ -241,19 +241,11 @@ export const findFtuUrlById = (ftuPartsArray, searchId) => {
   }
 
   // Find match
-  const foundMatch = ftuPartsArray.find((part) => {
-    const ftuIri = part.ftu_iri?.value;
-    const ftuPartIri = part.ftu_part_iri?.value;
-
-    // Check if either IRI value exists and includes the searchId
-    const isMatch =
-      (ftuIri && ftuIri.includes(searchId)) ||
-      (ftuPartIri && ftuPartIri.includes(searchId));
-
-    return isMatch;
-  });
+  const foundMatch = ftuPartsArray.find((ftuPart) =>
+    ftuPart.ftu_iri.includes(searchId) || ftuPart.ftu_part_iri.includes(searchId)
```

Comment:
Pushed a small change here. The 'part' isn't the purl url, it's just the id: (e.g.) CL_0009089. When searching against iri (e.g. "http://purl.obolibrary.org/obo/CL_0009089"), we need includes to match.